python/pylibcudf/pyproject.toml

```diff
@@ -39,11 +39,13 @@ classifiers = [
 
 [project.optional-dependencies]
 test = [
+    "cupy-cuda12x>=12.0.0",
     "fastavro>=0.22.9",
     "hypothesis>=6.131.7",
     "mmh3",
     "nanoarrow",
-    "numpy>=1.23,<3.0a0",
+    "numba-cuda>=0.18.0,<0.19.0a0",
```

Comment:
Tests are failing without `cuda.core`. This probably needs to be `numba-cuda[cu12]` to include the `cuda.core` dependency.

Reply:
Ugh thanks.